src/eduid_common/session/eduid_session.py

```diff
@@ -199,6 +236,9 @@ def __init__(self, config):
     def open_session(self, app, request):
         """
         See flask.session.SessionInterface
+
+        :return session
+        :rtype EduidSession
```

Comment:
I think rtype should be replaced by -> EduidSession, and I'm not sure I think we should redundantly document the obvious (":return (missing a colon) session")

Reply:
You can have:
def open_session(self, app, request) -> EduidSession:  # type: ignore

:)

Reply:
I also remove the old type hints as they didn't help.